meinberlin/urls.py

```diff
@@ -11,6 +11,7 @@
     url(r'^django-admin/', include(admin.site.urls)),
 
     url(r'^admin/', include(wagtailadmin_urls)),
+    url(r'^accounts/', include('allauth.urls')),
```

Comment:
I think I would prefer to to have them on top level, e.g. `/login` instead of `/accounts/login`.